library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -150,6 +151,19 @@ private void createDownloadsByBatchView(SQLiteDatabase db) {
         );
     }
 
+    private void createDownloadsByStatusView(SQLiteDatabase db) {
+        db.execSQL("DROP VIEW IF EXISTS " + DownloadContract.DownloadsByStatus.VIEW_NAME_DOWNLOADS_BY_STATUS);
```

Comment:
Technically this is not really `_BY_STATUS` if we take a look at the other view it has a different semantic as you don't pass the status and get the downloads for that status. Maybe this is more like `VIEW_NAME_DOWNLOADS_CHANGING_STATUS`, I don't know, this is just a restricted projection that doesn't include `total_bytes` and `current_bytes`.